src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -206,4 +208,58 @@ class BagsServletSpec extends TestSupportFixture
       body shouldBe s"Item $itemId not found"
     }
   }
+
+  it should "fail when done on an inactive/ hidden bag" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID") {
+      status shouldBe 410
+      body.lines.mkString should include(s"Tried to retrieve an inactive bag: $bagID")
```

Comment:
why use `should include(...)` instead of `shouldBe`? I'd much rather test for the whole message if that's possible.